.github/workflows/ci-it.yaml

```diff
@@ -49,3 +49,49 @@ jobs:
           CODECOV_TOKEN="d2065307-8f01-4637-9715-2781ef096db7" bash <(curl -s https://codecov.io/bash)
       - name: 'Check Dependencies Licenses'
         run: tools/dependencies/check-LICENSE.sh
+
+  CI-on-Windows:
+    runs-on: indows-latest
```

Comment:
What an odd name, is it a typo or really what GitHub provides?

Reply:
seems a typo

Reply:
Fixed.